### ISSUE TITLE
Transfer amounts don't work properly on buckets

### ISSUE THREAD

**Issue by TimrodDX** [space-wizards/space-station-14#5166]
## Description

How to reproduce:

- Take a bucket and set the transfer amount to 100 units.
- Go to a water tank and attempt to fill the bucket. It will only fill 10 units at a time.
- Curse your existence, as you'd rather be fetching creatures than these crummy pails of water.

![image](https://user-images.githubusercontent.com/7907891/140440778-1ad9e90e-8630-45a7-9832-1f6837f4a586.png)

![image](https://user-images.githubusercontent.com/7907891/140440793-0d8af52d-3cdb-441b-bb38-6fbee7120af4.png)

![image](https://user-images.githubusercontent.com/7907891/140440824-826a53d0-b35e-4be2-99dd-bdb31ca8288f.png)

**Comment by mirrorcult** [comment]
the issue isnt the bucket, its the water tank i imagine

**Comment by SweptWasTaken** [comment]
Yeah i'm pretty sure transfer amounts are for transferring *out* of the bucket not into it. Maybe add a transfer amounts verb to water tanks so you could like adjust the tap for how much came out?

**Comment by SweptWasTaken** [comment]
ReagentTank has a set transferAmount of ten.

https://github.com/space-wizards/space-station-14/blob/1508efff5410aa187e00a960021930295fbd9f0e/Content.Server/Chemistry/Components/ReagentTankComponent.cs#L17

Quick fix would be to just change this to 1000 but honestly you'd probably want to make that transferAmount linked to a SolutionTransfer component.

**Comment by AJCM-git** [comment]
related to #3926
